technical-reports/format/design-token.md

```diff
@@ -178,3 +178,36 @@ Tool vendors are encouraged to publicly share specifications of their extension
 <p class="ednote" title="Extensions section">
   The extensions section is not limited to vendors. All token users can add additional data in this section for their own purposes.
 </p>
+
+### Deprecated
+
+The optional **`$deprecated`** property is a boolean or string where tools MAY specify deprecated tokens. A token MAY be marked deprecated in any of the following scenarios:
+
+- A future update to the design system will remove this token
+- Removing the token now may break existing support
+- This token is discouraged from future use
+
+<aside class="example">
+
+```json
+{
+  "Button background": {
+    "$value": "#777777",
+    "$type": "color",
+    "$deprecated": true
+  },
+  "Button focus": {
+    "$value": "#70c0ff",
+    "$type": "color",
+    "$deprecated": "Please use {button.activeBorder} instead."
```

Comment:
Would downstream tooling need to convert `{button.activeBorder}` to a language-specific name?  This syntax is typically used for aliasing, and there's no clear instruction on:

1. Should this alias be resolved?
    1.  If so, what value should it resolve to?

Reply:
Yeah that’s a good question. I could see this being converted to, say, JSDoc or a CSS comment like so:

```tsx
  /** @deprecated Please use {button.activeBorder} instead */
  'Button focus': '#70c0ff',
```

In which case, would that be clear to people? The simplest interpretation is “the designers/dev write this message in a way that is clearest to their team” with no official guidance. I’m generally against parsing strings in any way, but yeah to your point there is a good question of “could this be improved?”

Nothing comes to mind, because in many cases there is no 1:1 replacement; it’s often a mixture of things. Perhaps we just adjust this example to not mention other tokens to avoid confusion?